python/sbp/client/drivers/network_drivers.py

```diff
@@ -88,8 +88,8 @@ def write(self, s):
 class HTTPException(Exception):
   pass
 
-DEFAULT_CONNECT_TIMEOUT = 10
-DEFAULT_READ_TIMEOUT = 10
+DEFAULT_CONNECT_TIMEOUT = 30
+DEFAULT_READ_TIMEOUT = 120
 DEFAULT_TIMEOUT = (DEFAULT_CONNECT_TIMEOUT, DEFAULT_READ_TIMEOUT)
 MAX_CONNECT_RETRIES = 10
```

Comment:
perhaps we should reduce the number of retries given the timeout change?  I think in this case we are talking 10s of minutes that requests will keep spinning.